unfoldingWord/translationCore
# Resetting alignment reducer data

# This pull request addresses:
- #2881 
- #2827 

# How to test this pull request:
- Open a Titus project with the `wordAlignment` tool
- Drag a word to the dropped zone.
- Then go to the home screen page and click on until you get to the `tools`  page again and click launch on the `wordAlignment` tool. 
(You should not get any errors)

# Other test
- Open a Titus project with the `wordAlignment` tool
- Drag a word to the dropped zone.
- Then go to the home screen page and import a different project. 
(You should not get any errors)


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/2894)
